pom.xml

```diff
@@ -250,8 +250,8 @@
     </dependency>
     <dependency>
       <groupId>com.laserfiche</groupId>
-      <artifactId>com.laserfiche.api.client.core</artifactId>
-      <version>1.0.1-SNAPSHOT</version>
+      <artifactId>lf-api-client-core</artifactId>
+      <version>1.0.0-preview-2762841586-SNAPSHOT</version>
```

Comment:
Does 2762841586 correspond to something meaningful?

Reply:
it represents the most recent preview snapshot version, which i will have to update again after my pending java client core PR gets merged in

Reply:
ok done